SwiftIntro.playground/Pages/Untitled Page.xcplaygroundpage/Contents.swift

```diff
@@ -228,49 +303,67 @@ var arrayOfItems: [Item] = [
 ]
 
 //create items names array of Strings from arrayOfItems use map is possible
+var namesOfItems: [String] = arrayOfItems.map { $0.name }
+namesOfItems.forEach { print($0) }
 
-//var namesOfItems: [String] =
-
-//namesOfItems.forEach { print($0) }
-
+//MARK: - NEED HELP >>>>>>>>>>
 //Create function that transforms value from dictionary into array of strings - description about movie (Good > 5, bad <=5
 //["Home Alone 1 (Good)", "Star Wars (Bad)", "Harry Potter (Good)", "Home Alone 4 (Bad)"]
 
 let movies: [String: Float] = ["Home Alone 1": 10, "Star wars": 5, "Harry Potter": 8, "Home Alone 4": 2]
 
-//let mappedMovies: [String]
+func transformValueToString(dict: [String: Float]) {
+//    if (dict.map { $0.value }) > 5 {
+//
+//    } else if (dict.map { $0.value }) <= 5 {
+//
+//    }
+}
```

Comment:
You can use map function and already pass block of code to your map. As I see from your code, you try to wrap 2 maps under 1 function, while you can fit all the logic under your map. I will show you an example and try to do it once again:)
```suggestion
let mappedMovies: [String] = movies.map { movie in
//    if movie.. {
//        return movie name and evaluation
//    } else {
//        return movie name and evaluation
//    }
}
try to solve it using this approach, if something is not clear, you can comment under here:)
```